src/components/Nav.jsx

```diff
@@ -7,6 +7,15 @@ function Nav() {
         <li>
           <Link to="/">Home</Link>
         </li>
+        <li>
```

Comment:
Think about a way you could generate this list more dynamically, it would help you to future-proof your app if more topics were added 😄